src/tape/ensemble.py

```diff
@@ -433,6 +433,28 @@ def check_lightcurve_cohesion(self):
                 return False
         return True
 
+    def sort_lightcurves(self):
```

Comment:
Could we please make sorting key (everything going after `id_col`) to be an argument of this function? I can easily imagine a need to sort by `[id_col, band_col, time_col]` or something like that

Reply:
It's definitely a good call to support sorting by band, but I decided not to add support for a fully general sorting key for now, and I decided to add a `by_band` parameter instead.

By just adding the `by_band` parameter, we can avoid users accidentally doing something like `[id_col, time_col, band_col]` which in general wouldn't be a sensible result. Of course users can still call `sort_values` themselves to sort arbitrary columns, but here we provide a simple helper function for the most common use cases.  Let me know your thoughts.

Reply:
Actually `[id_col, time_col, band_col]` also has some sense, for example for binned light curve. But I see your point, I'm fine with what we have now